src/components/grid-list/grid-tile.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Component,
+  ViewEncapsulation,
+  Renderer,
+  ElementRef,
+  Input,
+} from '@angular/core';
+
+import {coerceToNumber} from './grid-list';
+
+@Component({
+  selector: 'md-grid-tile',
+  host: { 'role': 'listitem' },
+  templateUrl: './components/grid-list/grid-tile.html',
+  styleUrls: ['./components/grid-list/grid-list.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class MdGridTile {
+  _rowspan: number = 1;
+  _colspan: number = 1;
+  _element: HTMLElement;
+  _isInitialized: boolean = false;
+
+  constructor(private _renderer: Renderer, element: ElementRef) {
+    this._element = element.nativeElement;
+  }
+
+  @Input('rowspan')
+  get rowspan() {
+  return this._rowspan;
+  }
+
+  @Input('colspan')
+  get colspan() {
+  return this._colspan;
+  }
+
+  set rowspan(value) {
+    this._rowspan = coerceToNumber(value);
+  }
+
+  set colspan(value) {
+    this._colspan = coerceToNumber(value);
+  }
+
+  setStyle(property: string, value: string): void {
```

Comment:
This seems like it should also be marked as `@internal`. I think it would also benefit from a comment explaining why styles need to be directly set.